app/components/blacklight/search/facet_suggest_input.html.erb

```diff
@@ -0,0 +1,9 @@
+<label for="facet-suggest-<%= facet.key %>">
+  <%= I18n.t('blacklight.search.facets.suggest.label', field_label: presenter&.label) %>
+</label>
+<input class="facet-suggest form-control"
```

Comment:
Should we use e.g. `text_field_tag` to make sure the attribute values get appropriately escaped?